source/console/minio-console.rst

```diff
@@ -131,316 +131,413 @@ routing rules may require setting a static MinIO Console port. For example,
+
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+   .. tab-item:: Watch
+
+      .. image:: /images/minio-console/console-watch.png
+         :width: 600px
+         :alt: MinIO Console Watch
+         :align: center
+
+      The :guilabel:`Watch` section displays S3 events as they occur on the selected bucket. 
+      This section provides similar functionality to :mc:`mc watch`.
 
-The :guilabel:`Heal` section displays the healing status for a bucket. 
-MinIO automatically heals objects and drives when it detects problems, such
-as drive-level corruption or a replacement drive.
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
 
-MinIO does not recommend performing manual healing unless explicitly directed
-by support. 
+   .. tab-item:: Drives
 
-This tab or its contents may
-not be visible if the authenticated user does not have the 
-:ref:`required administrative permissions <minio-policy-mc-admin-actions>`
+      .. image:: /images/minio-console/console-health.png
+         :width: 600px
+         :alt: MinIO Console Healing
+         :align: center
 
-Tools: Diagnostics
-------------------
+      The :guilabel:`Heal` section displays the healing status for a bucket. 
+      MinIO automatically heals objects and drives when it detects problems, such as drive-level corruption or a replacement drive.
 
-.. image:: /images/minio-console/console-diagnostics.png
+      MinIO does not recommend performing manual healing unless explicitly directed
+      by support. 
+
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+Support
+-------
+
+The :guilabel:`Support` section provides an interface for generating health and performance reports.
+You can also register your deployment with MinIO |subnet| to allow upload of health reports directly through the MinIO Console.
+
+This section contains the following subsections.
+Some subsections may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+.. tab-set::
+
+   .. tab-item:: Register
+
+      .. image:: /images/minio-console/console-register.png
+         :width: 600px
+         :alt: MinIO Console - SUBNET Registration login form
+         :align: center
+
+      The :guilabel:`Register` section provides a login form for connecting your MinIO deployment to |subnet|.
+
+      You can then upload your deployment health reports directly to SUBNET for reference by MinIO Engineering.
+
+      This section or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
```

Comment:
You already said this for this section in line 391. Delete that one or this one.